docs/decisions/006-default-style.md

```diff
@@ -0,0 +1,101 @@
+# Apply ECL 3 styling to default HTML tags
+
+| Status        | proposed                                              |
+| ------------- | ----------------------------------------------------- |
+| **Proposed**  | 12/10/2020                                            |
+| **Accepted**  | --/--/----                                            |
+| **Driver**    | @emeryro                                              |
+| **Approver**  |                                                       |
+| **Consulted** | @kalinchernev, @planctus                              |
+| **Informed**  | [@team](https://github.com/orgs/ec-europa/teams/inno) |
+
+## Decision
+
+[To be added]
+
+## Context
+
+- Styling of default HTML tags (without CSS class) has been a redundant request from some ECL users
+- In ECL 2, package `ecl-editor` was provided as a workaround. It worked quite well, but ended up raising some CSS conflict issues like [FRONT-1697](https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-1697)
+- In ECL 3 we shoud come with a more robust way to handle this, without disturbing the recommanded way to use ECL (by adding CSS classes)
+- We have no control over the final markup of pages. So we can't rely on any markup based selectors (like `>`, `+`, ...)
+
+## Requirements
+
+- This default styling should be made available as an optional CSS, or in an optional preset
+- The provided solution should be able to handle every HTML tag without having to update the code for each new component/utility
+
+## Consequences
+
+[To be added]
+
+## Alternatives Considered
+
+### Option 1: style elements directly
+
+**example:**
+
+- h1 { ... }
+- button { ... }
+
+**main pros:**
+
+- easy to override for component (very low specificity)
+
+**main cons:**
+
+- no easy way to prevent css leak to other elements on the page (Drupal admin menu, webtools widget, ...)
+
+This option is not robust enough and would surely ends up with unwanted side effects on the page.
+
+### Option 2: global namespace
+
+**example:**
```

Comment:
I believe that if we wrap this style definition into an ECL namespaced whatever then we reintroduce the same issue from another door, let's say.
Probably i don't entirely get the concerns, i mentioned info before and what we would have to do is something that has always been in place, there.

`h2:not([class*="ecl-"]) , .ecl-h2-class`

with one that extends the other, basically.

Sure this means that every definition of a style for a themed/un-themed h2 in a component that needs a different style would mean "overriding" a default rule, but i would also say that this could help us in building a consistent library, it's clear that there will not be any reason for h2s looking all different one from the other, for instance.

Reply:
Without a global namespace, the CSS would leak to other elements on the page (Drupal related stuff, webtools widgets, ...)
That's why I discarded this option

Reply:
But how could this be a concern for us? Webtools is usually working the other way around, their styles are often global and override ours. 
About Drupal, that's the intended behaviour, to be able to enforce some ecl style without the usage of specific markup, so this would be a feature, not a bug.

So, about these concerns, in the first case it's a matter of agreeing with someone on how to do things, the current situation is problematic already, the next one would still be if we don't all do the right thing:
ECL will define global styles for certain elements, injected styles by webtools should always be targeting their unique markup, it would be good for today, it will be good for tomorrow.

For Drupal, if there is any unwanted impact of these generic rules (i can't see it) we'll see, but how could this be the reason why we decide to drop the main, if not the only, option we have? This is what we always said "styling by default a limited set of html classes", if it has to be a themed markup again, i would not really understand why we wouldn't request the right markup.

Reply:
> Reading through comments here and briefly joining the meeting in the afternoon yesterday, I'm getting the impression about the following:
> 
> * Requirements are not fully clear. Specific use cases are necessary to be aligned with the platform
> * The direction we're going is more into creating a new [reset](https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/implementations/vanilla/packages/ec-preset-website/src/ec-preset-website.scss#L5) rather than a preset
> 
> Could it be that removing ecl-editor namespace and delivering the result in a separate CSS file serve the purpose?

@emeryro kalin's proposal is maybe a good solution, removing ecl-editor namespace and delivered it in a separate css file. we probably need to test the impact on the components, but maybe it will be minimal? what do you think?

Reply:
I'm still convinced that without namespace, we will run into troubles (and also we would produce some bad quality code).
But what we agree on is that we should deliver this extra css in a separated file